src/Validators/AllergyIntoleranceValidator.php

```diff
@@ -32,9 +32,10 @@ protected function configureValidator()
             self::DATABASE_INSERT_CONTEXT,
             function (Validator $context) {
                 $context->required('title')->lengthBetween(2, 255);
-                $context->required('begdate')->datetime('Y-m-d');
+                $context->optional('begdate')->datetime('Y-m-d H:i:s');
```

Comment:
Can you explain the reason for why you're making these fields be optional now?

Reply:
Hello! Oh yeah, forgot to mention that 🤦‍♂️ 
It's just the use case that worked for us, I made it optional because that's how the UI works, and also, it's something we needed in the API. It might be a good idea to leave it optional since it's optional in the UI, too. What do you think?

Reply:
So if you're just sending an begin date w/o an end date, this is working for you.
Since we're not very granular in our searches and we don't do paging in our returns the required end date was prob used to limit returns.
Though I suppose one could send an end date far in the future.

This doesn't bother me if okay with you Stephen.

Reply:
Also looks good to me

Reply:
High folks, I'm working with Ivan on this project via [NearForm](nearform.com), as well. Just wanted to chime in on this.

The reasoning behind not requiring a starting date for allergies was that most folks who report an allergy they have already been diagnosed with are unlikely to recall the exact date of diagnosis. Cardo's Head of Product, who is also a physician, preferred, in this scenario, having no starting date for allergies rather than an exact start date that was likely incorrect.

Reply:
So I'm fine with the change.  I started looking more closely at the datetime change and wondering if we will break existing clients.  It looks like Particle\Validator\Chain->datetime(..) forces the format so if any existing clients are using just the 'Y-m-d' format this will break those clients. 

I think that's fine and it should be a fairly small change for users to update the date format especially since that's what we are using now in the GUI.  We'll just need to mention it in the changelog/patch notes that this is a breaking change.